calculation/config.py

```diff
@@ -15,6 +15,8 @@
                     "update": "151203",
                     "replace": "151206",
                 },
+                'relevance': "True",
+                'condition': ".$_>1",
```

Comment:
is the "." on purpose (like to test this way of writing) ? (you put <100 in other)

Reply:
@delcroip @sniedzielski can we change `$_` for `INPUT`?

Reply:
![Screenshot from 2021-04-15 11-32-27](https://user-images.githubusercontent.com/52816247/114847715-4c19ec80-9dde-11eb-9232-cba7eae8682e.png) it is on wiki page that dot. I can quickly fix this

Reply:
@rstencel @delcroip No problem for me to change this $_ into INPUT

Reply:
No issue for me either